### PR TITLE
ncm-network: fix documentation bonding example

### PR DESCRIPTION
**Pull request by wdpypere**
The last commit is most relevant, this fixes the example to use ```bonding_opts``` instead of modules which does not work anymore.
The rest is a cleanup of this pod:
 - fixed pod errors
 - style adjustments for escaping special chars etc.
 - code blocks for examples instead of text
 - ...

### CODE REVIEW

File: ncm-network/src/main/perl/network.pod [hunk 1]
```diff
@@ -102,49 +134,52 @@ Setting nozeroconf to true stops an interface from being assigned an automatic a
...
 
-"/system/network/interfaces/bond0" = NETWORK_PARAMS;
-"/system/network/interfaces/bond0/driver" = "bonding";
-                                                                             
-include pro_software_component_modprobe;
-"/software/components/modprobe/modules" = push(nlist("name","bonding","alias","bond0")); 
-"/software/components/modprobe/modules" =
-push(nlist("name","bonding","options","mode=6 miimon=100"));
+    include 'components/modprobe/config';
+    "/software/components/modprobe/modules" = push(nlist("name","bonding","alias","bond0"));
```
Comment (ned21) [inline ncm-network/src/main/perl/network.pod]: push() is deprecated in favour of append()

File: ncm-network/src/main/perl/network.pod [hunk 2]
```diff
@@ -102,49 +134,52 @@ Setting nozeroconf to true stops an interface from being assigned an automatic a
...
 
-"/software/components/network/dependencies/pre" = push("modprobe");
+    "/software/components/network/dependencies/pre" = push("modprobe");
```
Comment (ned21) [inline ncm-network/src/main/perl/network.pod]: s/push/append/

File: ncm-network/src/main/perl/network.pod [hunk 3]
```diff
@@ -102,49 +134,52 @@ Setting nozeroconf to true stops an interface from being assigned an automatic a
...
+
+    "/system/network/interfaces/eth0/bootproto"="none";
+    "/system/network/interfaces/eth0/master"="bond0";
+
+    "/system/network/interfaces/eth1/bootproto"="none";
+    "/system/network/interfaces/eth1/master"="bond0";
 
-"/system/network/interfaces/eth1/bootproto"="none";
-"/system/network/interfaces/eth1/master"="bond0";
+    "/system/network/interfaces/bond0" = NETWORK_PARAMS;
```
Comment (ned21) [inline ncm-network/src/main/perl/network.pod]: The change from cushioned = (with spaces) from "uncushioned =" (without spaces) is a bit jarring. Would it be possible to clean this up and use the same consistent style?

**Comment by wdpypere** [comment]
fixed remarks from @ned21 .

File: ncm-network/src/main/perl/network.pod [hunk 4]
```diff
@@ -102,49 +134,52 @@ Setting nozeroconf to true stops an interface from being assigned an automatic a
...
 
 To enable channel bonding with quattor using devices eth0 and eth1 to form bond0, proceed as follows:
 
-include pro_software_component_network;
-                                                                             
-"/system/network/interfaces/eth0/bootproto"="none";
-"/system/network/interfaces/eth0/master"="bond0";
+    include 'components/network/config';
+
+    "/system/network/interfaces/eth0/bootproto" = "none";
```
Comment (stdweird) [inline ncm-network/src/main/perl/network.pod]: use prefix instead copying `/system/network/interfaces` over and over?

**Comment by wdpypere** [comment]
@stdweird fixed remarks

**Pull request merged**